src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -186,4 +188,16 @@ ParticipationStatus getParticipationStatus(
             @Path( STUDY_ID ) UUID studyId,
             @Path( PARTICIPANT_ID ) String participantId
     );
+
+    /**
+     * Retrieve questionnaire matching given entity key id
+     *
+     * @param studyId           - studyId
+     * @param questionnaireEKID - questionnaire entity key id
+     */
+    @GET( BASE + STUDY_ID_PATH + ENTITY_KEY_ID_PATH + QUESTIONNAIRE )
+    ChronicleQuestionnaire getChronicleQuestionnaire(
+            @Path( STUDY_ID ) UUID studyId,
+            @Path( ENTITY_KEY_ID ) UUID questionnaireEKID
+    );
```

Comment:
Let's swap `/questionnaire` with the id to get `/study/{studyId}/questionnaire/{questionnaireEKID}`, which is much more clear.

```
@GET( BASE + STUDY_ID_PATH + QUESTIONNAIRE + ENTITY_KEY_ID_PATH )
```